app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -115,12 +115,11 @@ fun HikeCard(
                           style = MaterialTheme.typography.bodySmall)
                       Text(
                           text =
-                              if (altitudeDifference == null)
+                              if (elevationGain == 0.0)
                                   stringResource(R.string.hike_card_no_data_label)
```

Comment:
Warning: If a hike has a real `elevationGain` of 0m we don't want to show "no_data". I think we should keep the convention of `elevationGain` to be null if we don't have any data.